var/spack/repos/builtin/packages/brion/package.py

```diff
@@ -19,7 +19,8 @@ class Brion(CMakePackage):
     version('3.1.0', tag='3.1.0', submodules=True)
     version('3.2.0', tag='3.2.0', submodules=True)
     version('3.3.0', tag='3.3.0', submodules=True)
-    version('3.3.1', tag='3.3.1', submodules=True, preferred=True)
+    version('3.3.1', tag='3.3.1', submodules=True)
+    version('3.3.2', tag='3.3.2', submodules=True, preferred=True)
```

Comment:
```suggestion
    version('3.3.2', tag='3.3.2', submodules=True)
```
Newest is always `preferred`.

Reply:
Thanks, updated